### PR TITLE
feat(styles): sets up root styles and linked stories

### LINKED ISSUE #5: Set up global themes
Using Tailwind as the base, update the global theme styles to fit Warren's design as closely as possible. We should use Tailwind extension/customisation to make it easy to apply to reskinned Tailwind components.

### PR DESCRIPTION
**Pull request by xanthipe**
**Issues affected**

Closes #5 

**Description**

* Finishes configuring the base for Tailwind
* Adds the starting points for light, dark and contrast themes
* Adds Storybook stories for colour and typography

**How to test**

Run up Storybook and check that the stories are present/the themes are switchable.

**Screenshots**

image
image

**Any other information?**

I've set this up using css variables rather than Tailwind's built in dark theme handling to make it easier to use multiple themes later. I've also removed Sass but implemented PostCSS nesting which is exactly the same syntax...

**Checklist**

- [x] Have all new components been added to Storybook?
- [ ] Have unit/component/integration tests been written?
- [x] Has documentation been updated?

### CODE REVIEW

**Comment by xanthipe** [comment]
Note: The Snyk extension is currently playing silly buggers but that isn't related to this PR. Please ignore it for now if it doesn't work for you and I'll pester @dantarian until we fix it...

File: src/themes/_colors.css
Comment (DiracLotus) [inline src/themes/_colors.css:1]: Are we preferring rgb or hex codes for defining colours? Happy either way (personal preference is hex)

Comment (xanthipe) [reply src/themes/_colors.css:1]: I'm easy either way as well, but @dantarian described about 2/3 of the site colours in RGB and it was fewer things to run through a translator.

File: src/themes/_colors.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+@layer base {
+    :root {
+        --primary: 173 6 6;
```
Comment (DiracLotus) [inline src/themes/_colors.css:3]: there are accessibility issues regarding our primary/primary-dark/error colours against the standard black text, as the colour contrast ratios are not high enough to meet AA guidelines https://webaim.org/resources/contrastchecker/

File: src/themes/_colors.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { FunctionComponent } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+const ColourBlocks: FunctionComponent = () => <div className='grid grid-cols-4 grid-rows-1 gap-1 p-2'>
+  <div className='flex h-20 items-center justify-center rounded border border-black bg-primary text-black'>primary</div>
+  <div className='flex h-20 items-center justify-center rounded border border-black bg-primary-dark text-black'>primary-dark</div>
```
Comment (DiracLotus) [inline src/themes/_colors.stories.tsx:6]: do we want an alternate text colour for display against primary-dark / use on dark modes?

Comment (xanthipe) [reply src/themes/_colors.stories.tsx:6]: See reply to previous - I was being lazy, I have learnt my lesson :)

File: src/themes/_colors.css (hunk 1 above)
Comment (xanthipe) [reply src/themes/_colors.css:3]: Huh, accessibility checker is now built in, that's an improvement. And here we are - one set of now AAA rated colour palettes. I'm expecting these to get tweaked by someone more designery as we go.

image
image
image

**Review by DiracLotus — APPROVED** [review]

**Pull request merged**